### ISSUE TITLE
*Compatibility Request: Chairs Reloaded*

### ISSUE THREAD

**Issue by pistols12** [msummers123/Movecraft-3#238]
Really was looking forward to chairs being added back into the compatible list for movecraft!!!! this possible?

**Comment by pistols12** [comment]
My Role play is serverly suffering without a captains chair being a real thing... : (

**Comment by FrozenBeard** [comment]
I totally agree, I use Chairs reloaded too, a lot of servers do, and would be amazing to "sit" while you cruise.

**Comment by keough99** [comment]
I would love to see movecraft support chairs plugins

**Comment by pistols12** [comment]
Like, how are we supposed to feel like captains of a ship without a
captain's chair!

**Comment by pistols12** [comment]
BUUUUUMP  :P
